static/js/legal/LegalSearch.js

```diff
@@ -9,24 +9,36 @@ const Pagination = require('./Pagination');
 class LegalSearch extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { q: $('#query').val(), from_hit: 0, advisory_opinions: []}
+    const initState = URI.parseQuery(window.location.search);
```

Comment:
No, it's `const` because the _reference_ to the object isn't changing. It would be `let` if I did `let initState = {obj1: 1}; initState = {obj2: 2}`, but it's `const` for `const initState = {obj1: 1}; initState.obj1 = 2`.